src/NuGet.Core/NuGet.Common/PublicAPI/net45/PublicAPI.Unshipped.txt

```diff
@@ -1,2 +1,4 @@
 NuGet.Common.NuGetLogCode.NU1010 = 1010 -> NuGet.Common.NuGetLogCode
 NuGet.Common.NuGetLogCode.NU1011 = 1011 -> NuGet.Common.NuGetLogCode
+NuGet.Common.TelemetryActivity.EndIntervalMeasure(string propertyName, System.Diagnostics.Stopwatch stopWatch) -> void
```

Comment:
What's the motivation here?
Because non overlapping ranges are not supported? 

Might be a good idea to move this to Common then?

https://github.com/NuGet/NuGet.Client/blob/ef00db94960b467d8318c8b3cd1760861c567eff/src/NuGet.Clients/NuGet.VisualStudio.Common/Telemetry/IntervalTracker.cs#L17

I'd wait on @zivkan's feedback for that though :)

Reply:
How the NuGetCore events can use the IntervalTracker?

Reply:
> What's the motivation here?
> Because non overlapping ranges are not supported?

Yes

Reply:
I don't think that `EndIntervalMeasure` is a good API name, because it has nothing to do with `StartIntervalMeasure`. `AddIntervalMeasure` would be more appropriate.